docs/adapting-new-services.md

```diff
@@ -58,7 +58,7 @@ Synchronous requests are ones where a user has made a call to Harmony and the co
 
 #### For Docker services
 
-Once complete, a service must send an HTTP POST request to the URL provided in the `callback` field of the Harmony input.  Failing to do so will cause user requests to hang until a timeout that is likely long in order to accommodate large, synchronous operations.  Please be mindful of this and provide ample error handling.
+Once complete, a service must send an HTTP POST request to the URL provided in the `callback` field of the Harmony input.  Failing to do so will cause user requests to hang until a timeout that is likely long in order to accommodate large, synchronous operations. Please be mindful of this and provide ample error handling.
```

Comment:
This isn't true anymore, is it? The result handler sends the responses, right?

Reply:
Updated the callback section to clarify what is required now that we're using Argo for docker services.